configs/train_configs/dpo/mini.yaml

```diff
@@ -3,7 +3,7 @@ model_revision: main
 use_flash_attn: true
 gradient_checkpointing: true
 tokenizer_name: EleutherAI/pythia-14m
-use_slow_tokenizer: true
+use_slow_tokenizer: null
```

Comment:
?

Reply:
I get



```
[rank0]: ValueError: Tokenizer class GPTNeoXTokenizer does not exist or is not currently imported.
[rank0]:[W916 10:35:25.965015944 ProcessGroupNCCL.cpp:1168] Warning: WARNING: process group has NOT been destroyed before we destruct ProcessGroupNCCL. On normal program exit, the application should call destroy_process_group to ensure that any pending NCCL operations have finished in this process. In rare cases this process can exit before this point and block the progress of another member of the process group. This constraint has always been present,  but this warning has only been added since PyTorch 2.4 (function operator())
```